backend/src/api/interviews/interviews.api.ts

```diff
@@ -97,6 +112,58 @@ const initInterviewsApi: FastifyPluginAsync<Options> = async (
       rep.status(HttpCode.OK).send(categoryIds);
     },
   });
+
+  fastify.route({
+    method: HttpMethod.GET,
+    url: `${InterviewsApiPath.$ID}${InterviewsApiPath.NOTES}`,
+    schema: { params: interviewNotesGetAllParamsValidationSchema },
+    preHandler: checkHasPermissions(
+      'oneOf',
+      PermissionKey.MANAGE_INTERVIEW,
+      PermissionKey.MANAGE_INTERVIEWS,
+    ),
+    async handler(
+      req: FastifyRequest<{ Params: InterviewNoteCreateRequestParamsDto }>,
+      rep,
+    ) {
+      const { id: interviewId } = req.params;
+      const notes = await interviewNoteService.getAll(interviewId);
+
+      return rep.status(HttpCode.OK).send({ items: notes });
```

Comment:
I think it's better to return the data in the same format, as it is returned from service

Reply:
But our developing convention format with ...GetAll.. and ...GetAllItem... requires that

Reply:
I agree, but you could return `{ items: ... }` from the service
In my opinion it's better to do so, because if we'll have to change the returning data format, we should make changes only to service, not to api

Reply:
> I agree, but you could return `{ items: ... }` from the service In my opinion it's better to do so, because if we'll have to change the returning data format, we should make changes only to service, not to api

Good point